setup.sh

```diff
@@ -38,7 +38,7 @@ trap '_error_report $LINENO' ERR
 # Default to empty or default values, avoiding unbound variable errors.
 SKIP_TERRAFORM=${SKIP_TERRAFORM:-}
 SKIP_TRIGGERS=${SKIP_TRIGGERS:-}
-SKIP_AUTH=${SKIP_AUTH:-}
+SKIP_AUTH=${SKIP_AUTH:=true}
```

Comment:
Other defaulting is using `:-`. Is there a reason this needs the `:=` syntax?

Reply:
`:-` defaults the other values to null or unset (don't skip);  `:=` is setting SKIP_AUTH to true (please skip).

Reply:
I poked around to confirm my understanding, and for our purposes this seems semantically identical: https://stackoverflow.com/a/26899206

So we could just as well use `${SKIP_AUTH:-true}`.

Not important enough to reopen.